### DISCUSSION TITLE
Application

### DISCUSSION THREAD

**Discussion by GameboyVlol** [community/community#46453] [GitHub Education] [answered]
Hello! So recently I applied for GitHub Education but I’m not sure if it got accepted or not, is there a way to check? Thanks

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @GameboyVlol 

You need to login with your GitHub account here:

https://education.github.com/

**Reply by GameboyVlol** [reply]
Hello! So I’m logged in obviously so I sent my application a few days ago but my question is, how do I know if I got accepted or not?

**Reply by SilasRodrigues19** [reply]
I don't remember because it's been a while since I did this, but if you can access the link below, it's been accepted:

https://education.github.com/pack

**Comment by OgaDavid** [comment]
I think a mail would be sent to the mail you used to register. I used my school email so, after I registered, I got a mail about verifying my GitHub account and that was all.
